tests/gpu_dfcc/gpu_dfcc1/input.dat

```diff
@@ -0,0 +1,29 @@
+#! aug-cc-pvdz H2O Test DF-CCSD vs GPU-DF-CCSD
+
+sys.path.insert(0,'../../../.')
+
+import gpu_dfcc
```

Comment:
shouldn't need the `sys.path.insert` or the `import gpu_dfcc` lines.

Reply:
oh duh, I just copied that from the plugin.  removed!